assets/lokomotive-kubernetes/packet/flatcar-linux/kubernetes/calico-host-protection.yaml.tmpl

```diff
@@ -3,7 +3,7 @@ clusterCIDRs:
 - ${cidr}
 %{ endfor ~}
 managementCIDRs:
-%{ for cidr in management_cidrs ~}
+%{~ for cidr in management_cidrs }
```

Comment:
Was it broken before? How? Or is it a cosmetic change with no effect to the managementCIDRs effectively used? Can you please clarify? :)

Reply:
In this specific case there is no difference between `{~ }` and `{ ~}` AFAICT. The former deletes the newline on the preceding line and the latter deletes the newline on the same line. It *would* matter if the following line was indented, for example.

Reply:
The difference shows up, when there is no `management_cidrs` specified, then:
```yaml
managementCIDRs:


```
is produced, instead of:
```yaml
managementCIDRs:
```

Not a big difference, but I spotted it as a unnecessary new line when testing the template. Can drop this patch as well actually. 

Also added explanation in the commit message.

Reply:
Thanks!